src/lib.rs

```diff
@@ -250,6 +250,7 @@ pub use serenity;
 /// use poise::serenity_prelude as serenity;
 /// ```
 pub mod serenity_prelude {
+    pub use serenity::model::application::interaction::{Interaction, InteractionResponseType};
```

Comment:
Can you add the other types that have had same-named type aliases added, too? Those are, in addition to Interaction and InteractionResponseType: InteractionApplicationCommandCallbackDataFlags, InteractionType, MessageInteraction

https://github.com/serenity-rs/serenity/pull/1899/files#diff-50cf0c1ab14aa2f5fce5fe01f9f241f8131c383e8973875b2fc0f7e495f57f1cR170-R189

Reply:
Btw since this is a draft PR; would it better if I refrain from reviewing until the PR is done?

Reply:
`interaction::MessageFlags` would probably overload `channel::MessageFlags`.

The PR is done beside the prelude additions.

Reply:
I could add `interaction::MessageFlags` as `InteractionMessageFlags`.

Reply:
I already added it as InteractionResponseFlags in the most recent commit. You can rename to InteractionMessageFlags though if you prefer that name